[BUG] Tests failing due to missing parameter

**Describe the bug**

When running the test suite, e.g. via CI, or, as in my case, while building a Debian package for this library, the tests fail with the error message:

```
[...]
# github.com/gotk3/gotk3/gtk [github.com/gotk3/gotk3/gtk.test]
Error: gtk/gtk_test.go:873:17: not enough arguments in call to button.Emit
	have (string)
	want (string, glib.Type, ...interface{})
FAIL	github.com/gotk3/gotk3/gtk [build failed]
[...]
```

**To Reproduce**

Run tests, e.g. using `go test -v ./...`. See https://github.com/satta/gotk3/actions/runs/8170159224/job/22335727500 as an example with all details.

**Expected behavior**

Tests should succeed.

**Error messages**

```
[...]
# github.com/gotk3/gotk3/gtk [github.com/gotk3/gotk3/gtk.test]
Error: gtk/gtk_test.go:873:17: not enough arguments in call to button.Emit
	have (string)
	want (string, glib.Type, ...interface{})
FAIL	github.com/gotk3/gotk3/gtk [build failed]
[...]
```


**Environment:**

gtk3 version: 3.24.41
go version: 1.22
os: Debian unstable
 
**Additional context**

See my PR #918 for a fix.

--- a/gtk/gtk_test.go
+++ b/gtk/gtk_test.go
@@ -870,7 +870,7 @@ func TestBuilder(t *testing.T) {
 		},
 	})
 
-	go button.Emit("clicked")
+	go button.Emit("clicked", glib.TYPE_STRING)
 
 	select {
 	case <-done:

